wordle-web/src/components/KeyBoard.vue

```diff
@@ -1,7 +1,14 @@
 <template>
   <v-row class="justify-center" dense v-for="(key, i) in keyboardLetters" :key="i">
-    <v-col v-for="(letter, j) in key" :key="j">
-      <LetterButton :letter="letter" @click="letterClick(letter)" />
+    <v-col cols="auto" v-for="(letter, j) in key" :key="j">
+      <v-hover v-slot="{ isHovering, props }">
```

Comment:
Cool hover effect!

Reply:
Thank you, I was doing some research for the drop shadow and came across the v-hover component along with the isHovering property and added some extra effects with it.